.circleci/continue_config.yml

```diff
@@ -328,10 +329,11 @@ workflows:
                 - databricks
                 - redshift
                 - bigquery
-          filters:
-            branches:
-              only:
-                - main
+                - clickhouse-cloud
+          # filters:
```

Comment:
Probably want to un-do this prior to merge

Reply:
that's my intention - should have put "TODO" on there